src/app/GitCommands/Git/SubmoduleHelpers.cs

```diff
@@ -7,7 +7,7 @@ namespace GitCommands.Git
 {
     public static partial class SubmoduleHelpers
     {
-        [GeneratedRegex(@"diff --git [^/\s]+/(?<filenamea>.+)\s[^/\s]+/(?<filenameb>.+)", RegexOptions.ExplicitCapture)]
+        [GeneratedRegex(@"diff --git\s+[ab]/(?<filenamea>.+)\s+[ba]/(?<filenameb>.+)", RegexOptions.ExplicitCapture)]
```

Comment:
Something like the following can still fail:
diff --git a/path/with a/space/ b/another/path

Reply:
of course
I decided: Who wants to shoot themself in the foot, shall do!